assets/src/blocks/Articles/useArticlesFetch.js

```diff
@@ -1,8 +1,8 @@
 import { useState, useEffect } from '@wordpress/element';
 import { fetchJson } from '../../functions/fetchJson';
+import { addQueryArgs } from '../../functions/addQueryArgs';
```

Comment:
Wouldn't this actually increase the bundle size given that `wp` is an external and is not included in the pipeline?

Reply:
No, because we can dequeue `wp.url` for the front end, it's the only thing we use from it. https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/430/commits/9c606f0b3b14c9afd274e90f997071b155aa6408#diff-4baaf0cfa9f65dafcc2bed33ae43cbc221e00364aaa33ec6b6c71b49e56ea5c0L402

Reply:
Ah, got it, didn't see it had its own package 👍

Reply:
There's also `apiFetch` in files we use on the front end, but we actually don't use it there already, because of an issue with WPML.

Reply:
Like here https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/9c606f0b3b14c9afd274e90f997071b155aa6408/assets/src/blocks/Articles/useArticlesFetch.js#L41-L41

Reply:
So it does seem to increase the bundle size (by some totally negligible few bytes), in some files. Of course it doesn't matter but maybe we could add some bundle comparison tool in the future for before/after checks, some can be integrated to the CI pipeline.

Reply:
I made up for those bytes by removing some unused imports :smile:

Reply:
And of course `wp.url` is much bigger, 4.7K compressed